tests/src/unit-deserialization.cpp

```diff
@@ -332,6 +332,14 @@ TEST_CASE("deserialization")
         {
             CHECK_THROWS_WITH_AS("[\"foo\",1,2,3,false,{\"one\":1}"_json, "[json.exception.parse_error.101] parse error at line 1, column 29: syntax error while parsing array - unexpected end of input; expected ']'", json::parse_error&);
         }
+
+        SECTION("FILE*")
+        {
+            std::FILE* f = std::fopen("nonexisting_file", "r"); // NOLINT(cppcoreguidelines-owning-memory)
```

Comment:
Now that we don't use `errno` any longer, you're right.

Reply:
I didn't even see the `errno` version until I read some more of the email thread.  :)

Reply:
Now UBSAN complains, because the function is marked `NONNULL`: https://github.com/nlohmann/json/runs/7433214543?check_suite_focus=true#step:5:235

I'll revert.

Reply:
I suppose that's a good thing that it's marked that way.  :D

Reply:
Yes - it's been a while since I saw a tool complain about the violation of such an annotation.